src/routes/news/_news.js

```diff
@@ -0,0 +1,70 @@
+// Dates must be in format YYYY-MM-DD
+// title must be unique within same date
+
+const newsList = [
+    {
+        title: 'ABC',
+        file: '20200207-a.txt',
+        img: 'kolintang.jpg',
+        img_name: 'kolintang',
+        date: '2020-02-07',
+    },
+    {
+        title: 'DEF',
+        file: '20200811.txt',
+        img: 'sasando.jpg',
+        img_name: 'sasando',
+        date: '2020-08-11',
+    },
+    {
+        title: 'GHI',
+        file: '20201113.txt',
+        img: 'angklung.jpg',
+        img_name: 'angklung',
+        date: '2020-11-13',
+    },
+];
+
+const transform = (newsList) => {
```

Comment:
To make it clear, We can add some comment describing in what formats will this transform function returns